aws/table_aws_ssm_maintenance_window.go

```diff
@@ -0,0 +1,343 @@
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "schedule_offset",
+				Description: "The number of days to wait to run a Maintenance Window after the scheduled CRON expression date and time.",
+				Type:        proto.ColumnType_INT,
+			},
+			{
+				Name:        "targets",
+				Description: "The targets of Maintenance Window.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getMaintenanceWindowTargets,
+				Transform:   transform.FromField("Targets"),
+			},
+			{
+				Name:        "tasks",
+				Description: "The Tasks of Maintenance Window.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getMaintenanceWindowTasks,
+				Transform:   transform.FromField("Tasks"),
+			},
+			{
+				Name:        "modified_date",
+				Description: "The date the Maintenance Window was last modified.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Hydrate:     getAwsSSMMaintenanceWindow,
+			},
+			{
+				Name:        "next_execution_time",
+				Description: "The next time the maintenance window will actually run, taking into account any specified times for the Maintenance Window to become active or inactive.",
+				Type:        proto.ColumnType_DATETIME,
+			},
+
+			/// Standard columns for all tables
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+			},
+			{
+				Name:        "tags",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsSSMMaintenanceWindowTags,
+				Transform:   transform.FromField("TagList").Transform(ssmMaintenanceWindowTagListToTurbotTags),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsSSMMaintenanceWindowAkas,
+				Transform:   transform.FromValue(),
+			},
+		}),
+	}
+}
+
+//// HYDRATE FUNCTIONS
+
+func getAwsSSMMaintenanceWindow(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+	logger.Trace("getAwsSSMMaintenanceWindow")
+
+	// TODO put me in helper function
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+	var id string
+	if h.Item != nil {
+		i := h.Item.(*ssm.GetMaintenanceWindowOutput)
+		id = *i.WindowId
+	} else {
+		id = d.KeyColumnQuals["window_id"].GetStringValue()
+	}
+
+	// Create Session
+	svc, err := SsmService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	// Build the params
+	params := &ssm.GetMaintenanceWindowInput{
+		WindowId: &id,
+	}
+
+	// Get call
+	data, err := svc.GetMaintenanceWindow(params)
+	if err != nil {
+		logger.Debug("getAwsSSMMaintenanceWindow", "ERROR", err)
+		return nil, err
+	}
+
+	return data, nil
+}
```

Comment:
Generally we follow the below order
1. LIST FUNCTION
2. HYDRATE FUNCTIONS
3. TRANSFORM FUNCTIONS